lib/engine/train.rb

```diff
@@ -23,6 +23,7 @@ def initialize(name:, distance:, price:, index: 0, **opts)
       @available_on = opts[:available_on]
       @discount = opts[:discount]
       @multiplier = opts[:multiplier]
+      @hex_edge_limit = opts[:hex_edge_limit]
```

Comment:
these are not eneded anymore rgiht, yo ucan cleanup this file and we can merge

Reply:
Oops, silly. Done.